wp-dependency-installer.php

```diff
@@ -50,18 +65,24 @@ class WP_Dependency_Installer {
 		 *
 		 * @var array $notices
 		 */
-		protected $notices = [];
+		protected $notices;
 
 		/**
 		 * Singleton.
 		 */
 		public static function instance() {
-			static $instance = null;
-			if ( null === $instance ) {
-				$instance = new self();
+			if ( ! self::$instance ) {
+				self::$instance = new self();
 			}
+			return self::$instance;
+		}
 
-			return $instance;
+		/**
+		 * Private constructor.
+		 */
+		protected function __construct() {
```

Comment:
As this is now a true singleton, shouldn't the scope of the contructor be `private`?

Reply:
Currently all class variables are protected, I have only followed the current coding standard ...

_Do I have to change them all?_

Reply:
No just asking.